### PR TITLE
Add WoW values to tech values

### PR DESCRIPTION
**Pull request by vonElfvin**

### CODE REVIEW

**Comment by palmenhq** [comment]
@vonElfvin there's also @HedvigInsurance/tech that you could ping here :) to get the app folks involved

**Comment by vonElfvin** [comment]
> 🆒
> Should we include something about not neglecting quality or nah?

I'd say yes, at least in another section, like the one "As Hedvig developers, when developing, we:"

Although, maybe it could be included here as well like "give ourself wiggleroom/estimations that llow us to not neglect quality in the sprints"

File: values.md [hunk 1]
```diff
@@ -39,4 +39,11 @@
 - Aren't afraid to try new things, but do __evaluate__
 - Value tech that is battle-tested over tech that is new and shiny. Although we value it, that does not mean
   it's our only path
+  
+## As Hedvig developers, when working in sprints, we:
 
+- Try and finish up leftovers from previous sprints first
+- Work without arbitrary deadlines
```
Comment (styrelius) [inline values.md:46]: Maybe have some sort of explanation of what counts as an "arbitrary" deadline? 🤔

File: values.md [hunk 2]
```diff
@@ -39,4 +39,11 @@
...
+- Freeze or cut scope when a deadline is set
+- Increase estimations when scope increase
```
Comment (hedvigoscar) [inline values.md:48]: This line presupposes the use of estimations, which I think we should move away from.

File: values.md [hunk 3]
```diff
@@ -39,4 +39,17 @@
...
 - Value tech that is battle-tested over tech that is new and shiny. Although we value it, that does not mean
   it's our only path
+  
+## As Hedvig developers, when working in sprints, we:
 
+- Do not consider estimates as deadlines
+  - Estimates are a way for us to communicate an educated guess of time we'll spend, nothing else
+- Try and finish up leftovers from previous sprints first
+- Work without arbitrary deadlines
+  - Where a deadline is considered arbitrary if it without an external dependency
```
Comment (styrelius) [inline values.md:49]: Is there a word missing here? Maybe it should say "... is considered arbitrary if it lacks external dependencies" or "... is considered arbitrary if it has no external dependencies" or "... is considered arbitrary if there's no external dependency". 🤖

File: values.md [hunk 4]
```diff
@@ -39,4 +39,17 @@
 - Aren't afraid to try new things, but do __evaluate__
 - Value tech that is battle-tested over tech that is new and shiny. Although we value it, that does not mean
   it's our only path
+  
+## As Hedvig developers, when working in sprints, we:
 
+- Do not consider estimates as deadlines
+  - Estimates are a way for us to communicate an educated guess of time we'll spend, nothing else
+- Try and finish up leftovers from previous sprints first
```
Comment (gustaveen) [inline values.md:47]: In most cases I this is a good rule of thumb and I totally agree. But I think we should be able to prioritise more pressing features or bugs if we no longer value the the leftovers. That being said it shouldn't be half finished features but rather if there's standalone tasks that could be done separately.

File: values.md [hunk 5]
```diff
@@ -39,4 +39,17 @@
...
+- Work without arbitrary deadlines
+  - Where a deadline is considered arbitrary if there's no external dependency
+- Freeze or cut scope when a deadline is set
```
Comment (palmenhq) [inline values.md:50]: *if a deadline is set? 😄 "when" implies that we have to have deadlines

File: values.md [hunk 6]
```diff
@@ -39,4 +39,17 @@
...
+- Prefer to have business decisions and design done before starting any development
+- Avoid big bang releases 
+  - We rather split up projects into phases of smaller batches or deploy in small iterations
+- Do not jeopardise quality
+  - We rather cut scope, spend more time on the issue or, if applicable, include more developers
```
Comment (palmenhq) [inline values.md:55]: ❤️

File: values.md [hunk 7]
```diff
@@ -39,4 +39,17 @@
...
+  
+## As Hedvig developers, when working in sprints, we:
 
+- Do not consider estimates as deadlines
+  - Estimates are a way for us to communicate an educated guess of time we'll spend, nothing else
+- Try and finish up leftovers from previous sprints first
+- Work without arbitrary deadlines
+  - Where a deadline is considered arbitrary if there's no external dependency
+- Freeze or cut scope when a deadline is set
+- Prefer to have business decisions and design done before starting any development
```
Comment (palmenhq) [inline values.md:51]: I don't agree with this. To me it implies tech should be excluded from business decisions and design which i think is a step in the wrong direction. Rather, i think we should try to work together and take decisions together. Thoughts?

**Review by palmenhq — COMMENTED** [review]
Awesome!

File: values.md [hunk 8]
```diff
@@ -39,4 +39,17 @@
 - Aren't afraid to try new things, but do __evaluate__
 - Value tech that is battle-tested over tech that is new and shiny. Although we value it, that does not mean
   it's our only path
+  
+## As Hedvig developers, when working in sprints, we:
```
[the remark this answers is not in the document]
Comment (hedvigoscar) [reply values.md:43]: #nosprints

File: values.md [hunk 9]
```diff
@@ -39,4 +39,19 @@
...
+  - If we could not avoid them and still want to finish them
+- Work without deadlines
+  - Unless there is not way to avoid a deadline
+- Finish the MVP first if a deadline is set
+  - We finalise the delivery as a whole after
+- Want to have dependencies done before starting implementation
+- Split up projects into small batch deliveries
+  - To avoid big bang project releases 
+- Do not jeopardise quality
+  - We rather cut scope, spend more time on the issue or, if applicable, include more developers
```
Comment (AlexSJ1990) [inline values.md:57]: we would rather

**Pull request merged**